### DISCUSSION TITLE
MySql Support

### DISCUSSION THREAD

**Discussion by bobbyphilip** [hasura/graphql-engine#8627] [General] [answered]
From https://hasura.io/docs/latest/graphql/core/guides/mysql-preview/, I can see the pull request & preview image.
However the linked PR is now closed & I couldnt find  a reference to a new one.
Is this something still being worked on? Happy to test if so

**Comment by meetzaveri** [comment, marked as answer]
Hello,

We can confirm that it's not included in our tasks board (as of now), as we are not actively working on it. We would recommend you to create a github issue on this with subject of feature request and follow it. That would be a good starting point and our team would be aware of it for tracking it in the future. Thanks!

**Reply by geckotechcr** [reply]
@meetzaveri ,
Just wish to confirm my understanding:  Hasura's work on MySQL integration has stopped and it is not currently planned for any future release.   _I am not suggesting that you will not ever integrate, i am just confirming that there is no scheduled release date_
Thanks
Sean

**Comment by dameleney** [comment]
We will be showing a preview of our new MySQL support during the community call tomorrow be sure to check it out. This will be a new implementation using the GDC connector method and will support initially support query features such as relationships with support for mutations coming soon in the Beta and GA release.
https://hasura.io/community/community-call/

**Reply by meetzaveri** [reply]
@bobbyphilip  @geckotechcr ☝🏻
